### PR TITLE
Update to Agent.Intercept 2.0.1

### PR DESCRIPTION
**Pull request by abaranch**
#99

### CODE REVIEW

File: Src/DependencyCollector/NuGet/Package.nuspec [hunk 1]
```diff
@@ -17,11 +17,11 @@
     <dependencies>
       <group targetFramework="net40">
         <dependency id="Microsoft.ApplicationInsights" version="[$coresdkversion$]" />
-        <dependency id="Microsoft.ApplicationInsights.Agent.Intercept" version="1.2.1" />
+        <dependency id="Microsoft.ApplicationInsights.Agent.Intercept" version="2.0.1" />
```
Comment (SergeyKanzhelev) [inline Src/DependencyCollector/NuGet/Package.nuspec]: make dependency to 2.0.*? instead of precise version?

Comment (abaranch) [inline Src/DependencyCollector/NuGet/Package.nuspec]: It means the same >= 2.0.1. Precise version would be [2.0.1]

Comment (SergeyKanzhelev) [inline Src/DependencyCollector/NuGet/Package.nuspec]: oh, ok.

**Comment by SergeyKanzhelev** [comment]
:shipit:

**Comment by dnduffy** [comment]
:shipit:

**Pull request merged**